src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,37 @@
-import React from 'react';
+/* eslint-disable max-len */
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+interface Props {
```

Comment:
![image](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/87986955/7c1279bb-c65c-4406-b31e-7d117477b3b6)